src/download.rs

```diff
@@ -142,6 +145,17 @@ impl Summary {
             ..self
         }
     }
+
+    /// Set the summary's resumable.
+    pub fn set_resumable(&mut self, resumable: bool) {
+        self.resumable = resumable;
+    }
+
+    /// Get the summary's resumable.
+    #[must_use]
+    pub fn resumable(&self) -> bool {
```

Comment:
maybe `fn is_resumable` might be better here?

Reply:
I think `resumable` is good because the "-able"  already implies a capability (i.e. capable to resume or not).